CMakeLists.txt

```diff
@@ -323,7 +323,7 @@ target_compile_definitions(ocos_operators PRIVATE ${OCOS_COMPILE_DEFINITIONS})
 target_link_libraries(ocos_operators PRIVATE ${ocos_libraries})
 
 file(GLOB shared_TARGET_LIB_SRC "shared/lib/*.cc" "shared/lib/*.h")
-if(CMAKE_SYSTEM_NAME STREQUAL "Emscripten")
+if(NOT OCOS_ENABLE_STATIC_LIB AND CMAKE_SYSTEM_NAME STREQUAL "Emscripten")
```

Comment:
IIRC, there is only one static library target for Emscripten, so, those build flags are bad now?

Reply:
So, the following target is only for WASM build pass verification only.